lectures/getting_started_julia/julia_essentials.md

```diff
@@ -1210,16 +1212,16 @@ This can also be achieved by running the following Julia code:
 
 ```{code-cell} julia
 open("us_cities.txt", "w") do f
-  write(f,
-"new york: 8244910
-los angeles: 3819702
-chicago: 2707120
-houston: 2145146
-philadelphia: 1536471
-phoenix: 1469471
-san antonio: 1359758
-san diego: 1326179
-dallas: 1223229")
+    write(f,
```

Comment:
Oh this must be why the `us_cities.txt` got messed.  Lets try it with something slightly different.  Try
```
data = "new york: 8244910
los angeles: 3819702
chicago: 2707120
houston: 2145146
philadelphia: 1536471
phoenix: 1469471
san antonio: 1359758
san diego: 1326179
dallas: 1223229"
open("us_cities.txt", "w") do f
```

Or something like that.  See if it doesn't mess up the formatting?

Reply:
Great.  Does it work?  Formatting doesn't break the file and it loads correctly?